aws/sumologic/lambda/cloudfront-s3-logs-to-sumologic/test/cloudfront_test.js

```diff
@@ -19,10 +18,11 @@ describe("Sumo Logic S3 CloudFront Lambda", function() {
 
       AWS.mock("S3", "getObject", new Buffer(TEST_DATA.compressed.cloudfront_logs));
 
-      subject.handler(TEST_DATA.compressed.lambda_event, fake_context(function(err) {
-        assert.equal(err, undefined);
+      subject.handler(TEST_DATA.compressed.lambda_event, {}, function(error, result) {
```

Comment:
You can use that parameter ([docs](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-handler.html#nodejs-prog-model-handler-callback)), but we don't so it's probably sensible to remove it.